Error: Current page must be indicated using `aria-current="page"`
Error: Nothing in the requested component required `aria-expanded` be used
Error: Nothing in the requested component required `aria-haspopup` be used
Error: Styles were not requested
diff --git a/output/Basic Navigation with Current Page/4/53e5ed66-fbcc-456c-9698-bfc2d80c548b.html b/output/Basic Navigation with Current Page/4/53e5ed66-fbcc-456c-9698-bfc2d80c548b.html
index 047d919..ab592f0 100644
--- a/output/Basic Navigation with Current Page/4/53e5ed66-fbcc-456c-9698-bfc2d80c548b.html	
+++ b/output/Basic Navigation with Current Page/4/53e5ed66-fbcc-456c-9698-bfc2d80c548b.html	
@@ -2,7 +2,7 @@
   <ul>
     <li><a href="/">Home</a></li>
     <li>
-      <a href="/products" aria-expanded="true" aria-haspopup="true">Products</a>
+      <a href="/products" aria-current="page">Products</a>
       <ul>
         <li><a href="/products/frobs">Frobs</a></li>
         <li><a href="/products/thrips">Thrips</a></li>
@@ -10,9 +10,4 @@
     </li>
     <li><a href="/services">Services</a></li>
   </ul>
-</nav>
-<style>
-  nav ul ul {
-    display: block;
-  }
-</style>
\ No newline at end of file
+</nav>
\ No newline at end of file
